src/Clock.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
```

Comment:
Could use a helper function to avoid repeat that annoying syntax 😁 
```suggestion
const getTime = () => new Date().toUTCString().slice(-12, -4);

```

Reply:
@Rijo-J is right. I would adjust it, like `getFormattedDate`, but the direction is very very reasonable